### PR TITLE
Include the hash of latest message in the stats

### PR DESCRIPTION
**Pull request by 0o153**
Work in progress, untested!

### CODE REVIEW

**Comment by martindsq** [comment]
Nice! What's the deal with those return nil? Could they still happen with a workable repo, or they indicate that the repo is broken?

**Comment by 0o153** [comment]
@martindsq Yeah, them indicate a broken feed (eg. feed with no messages)

File: go-ssb-bindings/debug.go [hunk 1]
```diff
@@ -107,6 +108,19 @@ func ssbRepoStats() *C.char {
 	counts.Messages = rootSeq.Seq()
 	counts.Messages += 1 // 0-indexed (empty is -1)
 
+	lm, err := sbot.RootLog.Get(rootSeq)
+	if err != nil {
+		retErr = errors.Wrap(err, "RepoStats: could not get the last message hash")
+		return nil
+	}
+
+	lastMsg, ok := lm.(ssb.Message)
```
Comment (martindsq) [inline go-ssb-bindings/debug.go:117]: I get a compiler error here: `undefined: ssb`

File: go-ssb-bindings/debug.go [hunk 2]
```diff
@@ -107,6 +109,19 @@ func ssbRepoStats() *C.char {
 	counts.Messages = rootSeq.Seq()
 	counts.Messages += 1 // 0-indexed (empty is -1)
 
+	lm, err := sbot.RootLog.Get(rootSeq)
+	if err != nil {
+		retErr = errors.Wrap(err, "RepoStats: could not get the last message hash")
+		return nil
```
Comment (martindsq) [inline go-ssb-bindings/debug.go:115]: I think we shouldn't return nil here as we may have a valid feed count and/or message count. For instance, when we onboard a new user, feeds and messages are empty but it doesn't have a last message hash. This is way the unit tests are failing.

Comment (0o153) [reply go-ssb-bindings/debug.go]: Fixed ^w^

**Pull request closed without merging**